plugins/scaffolder-field-extensions/scaffolder-frontend-module-http-request-field/src/components/SelectFieldFromApi/SelectFieldFromApi.tsx

```diff
@@ -83,7 +97,10 @@ const SelectFieldFromApiComponent = (
       headers.Authorization = `Bearer ${props.token}`;
     }
     const params = new URLSearchParams(
-      renderOption(options.params, { parameters: formContext.formData }),
+      renderOption(options.params, {
```

Comment:
Do we document the parameters feature anywhere? I presume this allows this type of thing:

```
path: /myservice/users/${{ identity.profile.name }}
```